.ruby-version

```diff
@@ -1 +1 @@
-2.5.0
+2.5.1
```

Comment:
Why `2.5.1` > `2.5.3` or `2.6.0`? Or was this change accidental?

Reply:
Hm, yeah.  I'll remove that.  What version should we be using?

Reply:
You know what, finding that out will be my next project.